test/runner/requirements/constraints.txt

```diff
@@ -2,6 +2,5 @@ coverage >= 4.2, != 4.3.2 # features in 4.2+ required, avoid known bug in 4.3.2
 pywinrm >= 0.2.1 # 0.1.1 required, but 0.2.1 provides better performance
 pylint >= 1.5.3, < 1.7.0 # 1.4.1 adds JSON output, but 1.5.3 fixes bugs related to JSON output
 sphinx < 1.6 ; python_version < '2.7' # sphinx 1.6 and later require python 2.7 or later
-yamllint < 1.8 ; python_version < '2.7' # yamllint 1.8 and later require python 2.7 or later
```

Comment:
Instead of removing this, I believe it can be changed as follows:

```
yamllint != 1.8.0 ; python_version < '2.7' # yamllint 1.8.0 requires python 2.7+ while earlier/later versions do not
```

That will avoid the problematic version on python 2.6 without requiring a newer version to be installed. We try to keep the version restrictions as relaxed as possible.